src/c++/vdo/fake/linux/wait.h

```diff
@@ -14,15 +14,16 @@
 
 #include "thread-utils.h"
 
+/* Fields which are not used in VDO unit tests are excluded. */
 typedef struct wait_queue_head {
-	spinlock_t		lock;
+	spinlock_t          	lock;
 	struct list_head	head;
```

Comment:
This line and the later one (23/24) change whitespace for no reason; could you change it back? Diffs are long enough as it is.